spec/gemstash/preload_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "spec_helper"
+
+describe Gemstash::Preload do
+  let(:stubs) { Faraday::Adapter::Test::Stubs.new }
+  let(:http_client) { Gemstash::HTTPClient.new(Faraday.new {|builder| builder.adapter(:test, stubs) }) }
+  let(:latest_specs) do
+    to_marshaled_gzipped_bytes([["latest_gem", "1.0.0", ""]])
+  end
+  let(:full_specs) do
+    to_marshaled_gzipped_bytes([["latest_gem", "1.0.0", ""], ["other", "0.1.0", ""]])
```

Comment:
Yeah, the tests are more in the style of one vs two, the naming is just to cover the use of one or the other depending on the expectation, not checking multiple versions of the same gem as I do not think that is an issue to be handled here.

Also, the third value is empty basically because we are ignoring it when fetching the gem (we just build the url by gem name and version) so it is there just to comply with the expectation of what will be read from the specs file.